src/facilities/costCenters.js

```diff
@@ -0,0 +1,153 @@
+import {
+  formatCostCenterFromServer,
+  formatCostCenterToServer,
+  formatCostCenterFacilityFromServer
+} from '../utils/facilities';
+
+/**
+ * Module that provides access to cost centers, and helps manage
+ * the relationship between those cost centers and facilities
+ *
+ * @typicalname contxtSdk.facilities.costCenters
+ */
+/**
+ * @typedef {Object} CostCenter
+ * @param {string} createdAt ISO 8601 Extended Format date/time string
+ * @param {string} [description]
+ * @param {string} id UUID
+ * @param {string} name
+ * @param {string} organizationId UUID
+ * @param {string} updatedAt ISO 8601 Extended Format date/time string
+ */
+/**
+ * @typedef {Object} CostCenterFacility
+ * @param {string} costCenterId UUID
+ * @param {string} createdAt ISO 8601 Extended Format date/time string
+ * @param {number} facilityId
+ * @param {string} id UUID
+ * @param {string} updatedAt ISO 8601 Extended Format date/time string
+ */
+class CostCenters {
+  /**
+   * @param {Object} sdk An instance of the SDK so the module can communicate with other modules
+   * @param {Object} request An instance of the request module tied to this module's audience.
+   * @param {string} baseUrl The base URL provided by the parent module
+   */
+  constructor(sdk, request, baseUrl) {
+    this._baseUrl = baseUrl;
+    this._request = request;
+    this._sdk = sdk;
+  }
+
+  /**
+   * Adds a facility to a cost center
+   *
+   * API Endpoint: '/costcenters/:costCenterId/facilities/:facilityId'
+   * Method: POST
+   *
+   * @param {string} costCenterId UUID corresponding with a cost center facility
```

Comment:
I'm being nit-picky but I think this is just `cost center` and not `cost center facility`